### PR TITLE
Add a11y test plan to docs

### PR DESCRIPTION
**Pull request by kimberlythegeek**
Added test plan from #497 to docs

### CODE REVIEW

File: test/integration/pages/home.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Home(Base):
     _new_entry_locator = (By.CSS_SELECTOR,
                           'section menu '
                           'button.{}:nth-child(1)'.format(
-                            munged_class_name('button')))
+                              munged_class_name('button')))
```
Comment (jrbenny35) [inline test/integration/pages/home.py]: Any reason for this change?

**Comment by jrbenny35** [comment]
There are a lot of commits for a simple doc addition, any way you can reduce them?

File: docs/accessibility-test-plan.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Manual Test Plan
+
+## KEYBOARD NAVIGATION
```
Comment (jimporter) [inline docs/accessibility-test-plan.md]: Editorial nit: the all-caps on these heading isn't consistent with the rest of our docs. We generally use Title Case (or Sentence case; we should pick one and stick to it).

File: test/integration/pages/home.py (hunk 1 above)
Comment (jimporter) [reply test/integration/pages/home.py]: When looking at this and the changes below, the new indentations seem more consistent, so I'm happy with it.

File: docs/accessibility-test-plan.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## UX Design Considerations
+- Interactions available with a mouse are also available using a keyboard.
+- Information conveyed visually is also available by other means.
+  - Pop-ups and other state changes are announced to screen readers using appropriate ARIA attributes.
+  - Temporary on-screen events give users ample time to receive conveyed information.
+- Animations are not excessively flashy, as this can cause seizures in some users.
+
+## TOOLS AND RESOURCES
+Use a variety of testing tools to analyze pages:
+- aXe developer tools by Deque
```
Comment (pdehaan) [inline docs/accessibility-test-plan.md]: Is this just the Firefox/Chrome add-on, a la https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/, or were you using some other aXe tools?

File: docs/accessibility-test-plan.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Browser developer tools to inspect HTML for logical heading structure and ARIA roles
+- [Color blind simulators](https://www.toptal.com/designers/colorfilter)
```
Comment (pdehaan) [inline docs/accessibility-test-plan.md]: Awesome resource, thanks!

**Comment by pdehaan** [comment]
> There are a lot of commits for a simple doc addition, any way you can reduce them?

When we merge this, somebody can press the "Squash and merge" button, which should reduce this down to 1 commit (without needing to do a messy rebase).

**Comment by pdehaan** [comment]
LGTM. I agree with @jimporter's comment about converting the H2 headings to use sentence case rather than all uppercase, for consistency.

File: test/integration/pages/home.py (hunk 1 above)
Comment (kimberlythegeek) [reply test/integration/pages/home.py]: I didn't realize I included these changes. I was only trying to push the new documentation file.

I got under-indentation errors when running flake8, which is the reason for these edits.

File: docs/accessibility-test-plan.md (hunk 3 above)
Comment (kimberlythegeek) [reply docs/accessibility-test-plan.md]: There are the browser extensions, but also the axe-core API, which is used by those extensions.

I've been working on a python package that uses axe-core to run automated accessibility tests.

**Pull request merged**